api/api-doc/storage_service.json

```diff
@@ -104,6 +104,68 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/toppartitions/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Toppartitions query",
+               "type":"toppartitions_query_results",
+               "nickname":"toppartitions_generic",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"table_filters",
```

Comment:
then it should be table or cf, if you are adding keyspace then it's only table on not keyspace:table

Reply:
there can be multiples of both, you can specify a couple of tables, which are identified by "ks:cf" and a couple of whole keyspaces. For example, you can request two tables: "ks:t, ks:simple" and one keyspace "system". The result would be toppartitions for the two tables in ks keyspace and all tables in the system keyspace